cmd/acra-translator/grpc_api/service.go

```diff
@@ -71,7 +72,6 @@ func (service *TranslatorService) Encrypt(ctx context.Context, request *EncryptR
 
 	response, err := service.service.Encrypt(ctx, request.Data, request.ClientId, nil)
 	if err != nil {
-		base.APIEncryptionCounter.WithLabelValues(base.EncryptionTypeFail).Inc()
```

Comment:
this is legacy metric that may be used now. lets leave with todo to remove it after deprecation?